File: backend_atelier_couture/src/main/java/iam/couture/projet/backend_atelier_couture/entity/Mesures.java
package iam.couture.projet.backend_atelier_couture.entity;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
@Table(name = "mesures")
public class Mesures {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    protected Long id;
    protected Float tete;
    protected Float cou;
    protected Float lbras;
    protected Float coude;
    protected Float genou;
    protected Float lcorps;
    protected Float cuisse;
    protected Float hanches;
    protected Float cheville;
    protected Float biceps;
    protected Float avantBras;
    protected Float poignet;
    protected Float entreJambes;
    protected Float epaules;
    protected Float coutureExt;
    protected Float htotale;
    protected  Float poitrine;
    @ManyToOne
    protected Client client;

}

File: angular-frontend/src/app/mesures.ts
import { Client } from './client';

export class Mesures {
  constructor(
    public id: number,
    public tete: number,
    public cou: number,
    public lbras: number,
    public coude: number,
    public genou: number,
    public lcorps: number,
    public ldos: number,
    public cuisse: number,
    public hanches: number,
    public cheville: number,
    public biceps: number,
    public avantBras: number,
    public poignet: number,
    public entreJambes: number,
    public epaules: number,
    public coutureExt: number,
    public creteIliaques: number,
    public dessousPoids: number,
    public htotale: number,
    public poitrine: number,
    public client: Client
  ) {}
}
